chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/CertifiedProductDownloadableResourceCreatorJob.java

```diff
@@ -194,6 +199,18 @@ private void swapFiles() throws IOException {
         } else {
             LOGGER.warn("Temp XML File was null and could not be moved.");
         }
+
+        if (tempJsonFile != null) {
+            String jsonFilename = getFileName(downloadFolder.getAbsolutePath(),
+                    getFilenameTimestampFormat().format(new Date()), "json");
+            LOGGER.info("Moving " + tempJsonFile.getAbsolutePath() + " to " + tempJsonFile);
+            Path targetFile = Files.move(tempJsonFile.toPath(), Paths.get(jsonFilename), StandardCopyOption.ATOMIC_MOVE);
```

Comment:
Please add a change to the Maintenance Plan in the chpl-build project to clean up the JSON files for the last N days. We do this with the generated XML files already and I think it could just be a few additional lines in that script.